result/convergence.yaml

```diff
@@ -0,0 +1,102 @@
+components:
+  schemas:
+    Convergence.Request:
+      description: >-
+        The container for a CPDP convergence request.
+      type: object
+      properties:
+        flow_names:
+          description: |-
+            Flow metrics will be retrieved for these flow names.
+            If no flow names are specified then all flows will be returned.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Flow/properties/name"
+          x-field-uid: 1
+    Convergence.Metric:
+      description: |-
+        The container for convergence measurements
```

Comment:
full stop. ( measurements or metrics ? )

Reply:
Changed to metrics, makes more sense.